src/native/mqtt5_client.c

```diff
@@ -1203,9 +1203,10 @@ JNIEXPORT void JNICALL Java_software_amazon_awssdk_crt_mqtt5_Mqtt5Client_mqtt5Cl
     return_data->java_client = java_client;
     return_data->jni_publish_future = (*env)->NewGlobalRef(env, jni_publish_future);
 
-    struct aws_mqtt5_publish_completion_options completion_options;
```

Comment:
`AWS_ZERO_STRUCT` can help, or your fix works as well.

Reply:
Good point. But since aws-crt-python and aws-crt-nodejs both use designated initializers for `completion_options` as well, I'll stick to it, just for consistency.